app/cells/datagrid_filter_cell.rb

```diff
@@ -0,0 +1,24 @@
+class DatagridFilterCell < FormCellBase
+  def show
+    case options[:filter].type
+    when :enum
+      cell(
+        :select,
+        nil,
+        select_options
+      ).call(:show)
```

Comment:
We already use plenty of view helpers in the cells. If it looks like a view, and quacks like a view... :hatched_chick: